src/hardwareModel.py

```diff
@@ -386,21 +416,41 @@ def gen_cacti_results(self):
         """
         buf_vals = cacti_util.gen_vals(
             "base_cache",
-            cacheSize=self.buffer_size, # TODO: Add in buffer sizing
+            cacheSize=2048, #self.buffer_size, 
             blockSize=64,
             cache_type="cache",
             bus_width=self.buffer_bus_width,
         )
         logger.info(f"Buffer cacti with: {self.buffer_size} bytes, {self.buffer_bus_width} bus width")
+        buf_opt = {
+            "ndwl": buf_vals["Ndwl"],
+            "ndbl": buf_vals["Ndbl"],
+            "nspd": buf_vals["Nspd"],
+            "ndcm": buf_vals["Ndcm"],
+            "ndsam1": buf_vals["Ndsam_level_1"],
+            "ndsam2": buf_vals["Ndsam_level_2"],
+            "repeater_spacing": buf_vals["Repeater spacing"],
+            "repeater_size": buf_vals["Repeater size"],
+        }
+
         mem_vals = cacti_util.gen_vals(
             "mem_cache",
-            cacheSize=self.mem_size,
+            cacheSize=131072,  # self.mem_size,
```

Comment:
We need to figure out why this issue arises. What mem_size, buf_size, bandwidth values are allowed. 
Not for this PR, but will be your next task big task.